Doc/tools/templates/customsourcelink.html

```diff
@@ -3,8 +3,14 @@
     <h3>{{ _('This Page') }}</h3>
     <ul class="this-page-menu">
       <li><a href="{{ pathto('bugs') }}">{% trans %}Report a Bug{% endtrans %}</a></li>
-      <li><a href="{{ pathto('_sources/' + sourcename, true)|e }}"
-            rel="nofollow">{{ _('Show Source') }}</a></li>
+      <li>
+        {% if version == "3.7" %}
```

Comment:
Thanks for the review, @methane :)
I removed the if statement as suggested.
Branches 2.7, 3.5, and 3.6 don't have the if statement anymore.